app/views/subscription/index.phtml

```diff
@@ -130,7 +130,7 @@
 			    data-feed-id="<?php echo $feed->id(); ?>"
 			    dropzone="move">
 				<a class="configure open-slider" href="<?php echo _url('subscription', 'feed', 'id', $feed->id()); ?>"><?php echo _i('configure'); ?></a>
-				<img class="favicon" src="<?php echo $feed->favicon(); ?>" alt="✇" /> <?php echo $feed->name(); ?>
+				<img class="favicon <?php if ($feed->mute()) {echo 'mute';}?>" src="<?php echo $feed->favicon(); ?>" alt="✇" /> <?php echo $feed->name(); ?>
```

Comment:
I think we should output the same parameters here, and also in:
* https://github.com/FreshRSS/FreshRSS/blob/63a8f5e167e4bf17f76eb6e8baa9b75814c4512d/app/layout/aside_feed.phtml#L56
* https://github.com/FreshRSS/FreshRSS/blob/63a8f5e167e4bf17f76eb6e8baa9b75814c4512d/app/views/index/global.phtml#L40

See the `data-priority` attribute

Reply:
I am not sure what you mean here. I'll need some more explanations

Reply:
We have 3 places with a similar construct, which I suggest to keep uniform.
A bit higher up, there is a line 128 which currently contains `<li class="item feed<?php echo $error, $empty; ?>"`.
I suggest to replace it with something along the lines of:
```php
<li class="item feed<?php echo $error, $empty, $feed->mute() ? 'mute' : ''; ?>" data-priority="<?php echo $feed->priority(); ?>"`
```
So that we keep all the feed properties at the same element level.
(And adjust your CSS rule below accordingly)

And do the same (i.e. add the mute info) to the two other referenced lines.

Reply:
Ok, I see what you mean.

Reply:
done